src/main/java/com/google/sps/data/ScheduledInterview.java

```diff
@@ -15,17 +15,24 @@
 package com.google.sps.data;
 
 import com.google.auto.value.AutoValue;
+import com.google.gson.Gson;
+import com.google.gson.annotations.SerializedName;
 
 /** Represents a scheduled interview. */
 @AutoValue
 public abstract class ScheduledInterview {
+  @SerializedName("id")
```

Comment:
Did you end up using these? If not, please remove them.

Reply:
Done